cmd/notation/verify.go

```diff
@@ -35,6 +35,19 @@ func verifyCommand(opts *verifyOpts) *cobra.Command {
 	command := &cobra.Command{
 		Use:   "verify [reference]",
 		Short: "Verifies OCI Artifacts",
+		Long: `
+** This command is in preview and under development. **
+Prerequiste: a trusted certificate needs to be generated or added using the command "notation cert". 
+
+Example - Verify a signature using the trusted certificate:
+  notation verify <registry>/<repository>:<tag>
+
+Example - Verify a signature associated with an OCI artifact identified by the digest:
+  notation verify <registry>/<repository>@<digest>
+
+Example - Verify a signature using a trusted certificate in a specified path:
+  notation verify --cert-file <cert_path> <registry>/<repository>:<tag>
```

Comment:
I would remove this example since we would onle support trust store based verificaiton,

Reply:
Agree